README.md

```diff
@@ -1,295 +1,61 @@
-# setup-python V4
+# setup-python
 
 <p align="left">
   <a href="https://github.com/actions/setup-python"><img alt="GitHub Actions status" src="https://github.com/actions/setup-python/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-This action sets up a Python environment for use in actions by:
+This action provides the following functionalities for GitHub Actions users:
 
-- optionally installing and adding to PATH a version of Python that is already installed in the tools cache.
-- downloading, installing and adding to PATH an available version of Python from GitHub Releases ([actions/python-versions](https://github.com/actions/python-versions/releases)) if a specific version is not available in the tools cache.
-- failing if a specific version of Python is not preinstalled or available for download.
-- optionally caching dependencies for pip, pipenv and poetry.
-- registering problem matchers for error output.
+- Optionally downloading and installing the requested version of Python/PyPy and adding it to the PATH
+- Optionally caching dependencies for pip, pipenv and poetry
+- Registering problem matchers for error output
 
```

Comment:
I'd like to keep all points because it should be like a syllabus for the action.